### PR TITLE
U1-81 Write unit test for action random-desktop

### PR DESCRIPTION
**Pull request by duyphaphach**
JIRA Ticket: https://pass-career.atlassian.net/browse/U1-81

### CODE REVIEW

File: source/actions/get-images.js [hunk 1]
```diff
@@ -67,6 +67,8 @@
     'Exclude duplicates',
     400,
   );
+  console.log('unique images');
+  console.log(uniqueImages);
```
Comment (knightus) [inline source/actions/get-images.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980094)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980094&bid=16448180)

File: source/actions/get-images.js [hunk 2]
```diff
@@ -67,6 +67,8 @@
     'Exclude duplicates',
     400,
   );
+  console.log('unique images');
```
Comment (knightus) [inline source/actions/get-images.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980094)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Coverage increased per file
===========================
+ source/actions/get-images.js  70
+ source/helpers/set-wallpaper.js  88
+ source/actions/random-desktop.js  71
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980094&bid=16448180)

File: source/actions/get-images.js [hunk 3]
```diff
@@ -67,6 +67,8 @@ export default async function (args, options, logger) {
...
+  console.log(uniqueImages);
```
Comment (knightus) [inline source/actions/get-images.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980094)

File: source/actions/get-images.js [hunk 4]
```diff
@@ -67,6 +67,8 @@ export default async function (args, options, logger) {
     'Exclude duplicates',
     400,
   );
+  console.log('unique images');
```
Comment (knightus) [inline source/actions/get-images.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980094)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ source/actions/get-images.js  68
+ source/helpers/set-wallpaper.js  88
+ source/actions/random-desktop.js  71
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980094&bid=16448180)

**Review by huyhqit — APPROVED** [review]

**Pull request merged**